clients/unshaded/src/main/java/tachyon/client/file/TachyonFileSystemUtils.java

```diff
@@ -134,4 +141,40 @@ public static boolean waitCompleted(final TachyonFileSystemCore tfs, final Tachy
 
     return completed;
   }
+
+  /**
+   * Persist the given file to the under file system
+   * @param file the file to persist
+   * @param dstPath the destination in the under file system
+   * @param tfs TachyonFileSystem to carry out tachyon operations
+   * @param tachyonConf TachyonConf object
+   * @return the size of the file persisted
+   * @throws IOException if an I/O error occurs
+   * @throws FileDoesNotExistException if the given file does not exist
+   * @throws TachyonException if an unexpected Tachyon error occurs
+   */
+  public static long persistFile(TachyonFile file, TachyonURI dstPath, TachyonFileSystem tfs,
```

Comment:
Is it possible to stay consistent with the other methods here, so that <code>tfs</code> is first, and it is of type <code>TachyonFileSystemCore</code>?

Also, is <code>dstPath</code> supposed to be related to <code>file</code>? Or is this utility supposed to allow any file to be written to any path (not necessarily the file's correct ufs path)?